prog/postgres/postgres_nexus.rb

```diff
@@ -0,0 +1,184 @@
+# frozen_string_literal: true
+
+require "forwardable"
+
+require_relative "../../lib/util"
+
+class Prog::Postgres::PostgresNexus < Prog::Base
+  extend Forwardable
+  def_delegators :server, :vm
+
+  semaphore :restart, :destroy
+
+  def self.assemble(project_id, location, server_name, vm_size, storage_size_gib)
```

Comment:
The `assemble` method for other resources use named parameters. Should we keep this convention?

```
self.assemble(public_key, project_id, name: nil, size: "standard-2",
    unix_user: "ubi", location: "hetzner-hel1", boot_image: "ubuntu-jammy",
    private_subnet_id: nil, nic_id: nil, storage_volumes: nil, boot_disk_index: 0,
    enable_ip4: false)
```

```
  def self.assemble(sshable_hostname, location: "hetzner-hel1", net6: nil, ndp_needed: false, provider: nil, hetzner_server_identifier: nil)
```

```
  def self.assemble(project_id, name: nil, location: "hetzner-hel1", ipv6_range: nil, ipv4_range: nil)
```

Reply:
It seems we use named parameters only for optional parameters, other assemble methods don't use named parameters for non optional parameters. None of the parameters in postgres_nexus assemble is optional.

I'm fine with making all parameters(optional/non-optional) named, but that should be a different PR probably.

Reply:
Makes sense